website/layouts/partials/_default/banner.html

```diff
@@ -0,0 +1,16 @@
+{{- $currentVersion := index (split .Dir "/") 1 }}
+{{- $latestVersion := .Site.Params.latest }}
+{{- $notLatestVersion := ne $currentVersion $latestVersion }}
+{{- $latestUrl := replace .URL $currentVersion $latestVersion }}
+<header>
+  <h1 class="is-size-1{{ if .Params.description }} is-spaced{{ end }}">{{ .Title }}</h1>
+  {{- with .Params.description }}<h4 class="subtitle is-size-4">{{ . }}</h4>{{ end }}
+
+  {{- if $notLatestVersion }}
```

Comment:
let's just put `ne $currentVersion $latestVersion ` here, no need for variable if used once (:

Reply:
Done